plugins/inputs/burrow/burrow.go

```diff
@@ -218,11 +219,21 @@ func (b *burrow) createClient() (*http.Client, error) {
 		return nil, err
 	}
 
+	var transport http.Transport
+	timeout := time.Duration(b.ResponseTimeout)
+	transport.DialContext = (&net.Dialer{
+		Timeout: timeout,
+		DualStack: true,
+	}).DialContext
+	transport.TLSClientConfig = tlsCfg
+	transport.MaxIdleConnsPerHost = b.ConcurrentConnections / 2
+	transport.MaxConnsPerHost = b.ConcurrentConnections
+	transport.ResponseHeaderTimeout = timeout
+	transport.IdleConnTimeout = 90*time.Second
```

Comment:
```suggestion
	timeout := time.Duration(b.ResponseTimeout)
	transport := http.Transport{
		DialContext: &net.Dialer{Timeout: timeout, DualStack: true}.DialContext,
		TLSClientConfig: tlsCfg,
		MaxIdleConnsPerHost: b.ConcurrentConnections / 2,
		MaxConnsPerHost: b.ConcurrentConnections,
		ResponseHeaderTimeout: timeout,
		IdleConnTimeout: 90*time.Second,
	}
```

One concern is that `MaxIdleConnsPerHost: b.ConcurrentConnections / 2` can become zero for `b.ConcurrentConnections` being zero or one. What happens in this case?

Reply:
As documentation says, if MaxIdleConnsPerHost is zero, then DefaultMaxIdleConnsPerHost is used (which is equal to `2`).
If ConcurrentConnections is zero, ie MaxConnsPerHost is zero, it means "no limits".

Reply:
Would you care to add this comment to the code so that the next reader doesn't have to look it up?